### ISSUE TITLE
Generated files imports are broken in 1.4.8

### ISSUE THREAD

**Issue by pomdtr** [NfNitLoop/deno-embedder#16]
Error message:

```
Unknown export './src/embed.ts' for '@nfnitloop/deno-embedder@1.4.8'.
```

`1.4.7` works fine.

**Comment by NfNitLoop** [comment]
Do you have a way to reproduce this? It's working fine for me.

**Comment by NfNitLoop** [comment]
Aha, found it.  Fixed in 85acb5811c0bffeb64553918ccb203f5cf75f803 and released 1.4.9.
